src/Log/LogService.php

```diff
@@ -353,19 +357,17 @@ protected function createLogLevelFilter($logLevel)
      */
     public function getLog($logName = null)
     {
-        if ($logName = null) {
-            return getDefaultLog();
-        }
-        if ($logName == 'default') {
+        if ($logName = null || $logName == 'default') {
             return $this->getDefaultLog();
         }
+
         if (array_key_exists($logName.'.log', $this->loggers)) {
             return $logger = $this->loggers[$logName.'.log'];
         } else {
             try {
                 return $this->createLog($logName);
-            } catch (Exception $e) {
-                throw new Exception('Unknown logger: '.$logName, 1);
+            } catch (\Exception $e) {
```

Comment:
What is our policy do we create new logs for any name provided or do we require a call to createLog for unknown loggers. I am leaning towards easier usage and simply creating new loggers. In any case the behavior needs to be documented. In the function comment and in tests.

Reply:
The initial idea for `getlog()` was to look for a log and create one if it already doesn't exist. Should we create logs for any name provided? I am not sure about it yet. For now, I am keeping it for simply creating new loggers. Will document the same.